backend/cmd/headlamp.go

```diff
@@ -384,9 +384,6 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 		r = baseRoute.PathPrefix(config.baseURL).Subrouter()
 	}
 
-	fmt.Println("*** Headlamp Server ***")
-	fmt.Println("  API Routers:")
-
```

Comment:
We want these prints.

Reply:
Ok, I reverted it and use the `logger.Log` function instead. But I don't understand the content of the log lines 🤔